dotnet/aspnet46/functionApp/ArmTemplates/windows-functionapp-template.json

```diff
@@ -0,0 +1,132 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "functionAppName": {
+            "type": "string",
+            "metadata": {
+            "description": "The name of the function app that you wish to create."
+            }
+        },
+        "sku": {
+            "type": "string",
+            "defaultValue": "Dynamic Y1",
+            "allowedValues": [
+                "Dynamic Y1",
+                "Standard S1"
+            ]
+        },
+        "storageAccountName": {
+            "type": "string"          
+        },  
+        "hostingPlanName": {
+            "type": "string"
+        }
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Storage/storageAccounts",
+            "sku": {
+                "name": "Standard_LRS",
+                "tier": "Standard"
+            },
+            "kind": "Storage",
+            "name": "[parameters('storageAccountName')]",
+            "apiVersion": "2018-07-01",
+            "location": "[resourceGroup().location]",
+            "tags": {},
+            "scale": null,
+            "properties": {
+                "networkAcls": {
+                    "bypass": "AzureServices",
+                    "virtualNetworkRules": [],
+                    "ipRules": [],
+                    "defaultAction": "Allow"
+                },
+                "supportsHttpsTrafficOnly": false,
+                "encryption": {
+                    "services": {
+                        "file": {
+                            "enabled": true
+                        },
+                        "blob": {
+                            "enabled": true
+                        }
+                    },
+                    "keySource": "Microsoft.Storage"
+                }
+            },
+            "dependsOn": []
+        },
+      {
+        "type": "Microsoft.Web/serverfarms",
```

Comment:
Is this the template for the consumption based hosted plan?

Reply:
Yes, the default is the consumption-based plan(Y1  Dynamic).